src/main/java/com/commercetools/sync/commons/utils/CustomUpdateActionUtils.java

```diff
@@ -342,6 +342,7 @@
         @Nonnull final Function<T, String> updateIdGetter) {
 
         return newCustomFields.keySet().stream()
+                              .filter(newCustomFieldName -> Objects.nonNull(newCustomFields.get(newCustomFieldName)))
```

Comment:
it would be nice if you update the Javadoc too, something like this: 
`<p>Note: Null custom fields are filtered out.`

Reply:
aec6bf09c83bd97530d157cf40ff923842896a37